XXX push upstream
../vte-0.70.1/src/dumpkeys.c:164:2: error: implicit declaration of function 'cfmakeraw' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
        cfmakeraw(&tcattr);

Index: meson.build
--- meson.build.orig
+++ meson.build
@@ -215,6 +215,7 @@ endif
 
 libc_feature_defines = [
   ['_GNU_SOURCE', '1'],
+  ['_BSD_SOURCE', '1'],
   ['_POSIX_C_SOURCE', '200809L'],
   ['_XOPEN_SOURCE', '700'],
   ['_XOPEN_SOURCE_EXTENDED', '1'],
